Xamarin.Forms.Core/Application.cs

```diff
@@ -206,6 +219,19 @@ public ResourceDictionary Resources
 
 		public event EventHandler<Page> PageDisappearing;
 
+		public static void SetFlags(params string[] flags)
```

Comment:
we already have the flags in Core at the Device level. Application, or Device: pick one

Reply:
D'oh. Somehow I missed that we already had them in Device.

Reply:
Yeah, that was dumb. Fixed.